transport/src/main/java/io/el/channel/AbstractChannel.java

```diff
@@ -73,11 +75,13 @@ protected abstract class AbstractInternal implements Internal {
     @Override
     public void register(ChannelEventLoop eventLoop, ChannelPromise promise) {
       // TODO: implement me
+      AbstractChannel.this.channelEventLoop = eventLoop;
```

Comment:
Is this temporary code right?

Reply:
Yes, it is temporary.
The channelEventLoop is required to run handler events.